scripts/label_all_tokens.py

```diff
@@ -32,10 +33,14 @@ def main():
         default="delphi-suite/delphi-llama2-100k",
         required=False,
     )
+    parser.add_argument(
+        "--output",
+        help="Output path name. Must include at least output file name.",
```

Comment:
I think it's either _output path_ or _output file name_, not output path name. This is a bit weird though, because: what if I want to save in my current working directory? I'll just pass "myfilename.pkl" meaning "./myfilename.pkl" but it'll save in static dir.

Reply:
I endorse both points.

On reflection, we should be *writing* to the static dir very rarely, and when we do it's reasonable to expect the user to move the files there and check them in. The main purpose of the static dir is to have a straightforward way to *read*.

So output paths should just use the path the user passes in the most straightforward way possible, without any reference to the static dir.

Reply:
Earlier in the code, line 60-69, there is a section that writes down `all_tokens_list.txt`, which the script isn't giving the user a choice on where to load in the script argument. Should I leave the code as-is, letting the script saving this to `static` or give it the same treatment as the labelled token pickle by adding an extra argument?

![image](https://github.com/delphi-suite/delphi/assets/62800649/b05883b3-da2e-4941-8e54-8f2965399d2a)